wiki/Skinning/Compendium/Anime/en.md

```diff
@@ -0,0 +1,238 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Anime
+Compendium of all completed anime osu! skins.
+
+- ![][o!s]  - means the skin contains osu!standard elements.
+- ![][o!c]  - means the skin contains osu!catch elements.
+- ![][o!t]  - means the skin contains osu!taiko elements.
+- ![][o!m]  - means the skin contains osu!mania elements.
+
+## A
+
+- [Abigail Williams](/community/forums/topics/808012) by [SleepingD](/users/9612894)![][o!s]
```

Comment:
move the mode icons to the front and add spaces between them

this line for example:
```suggestion
- ![][o!s] [Abigail Williams](/community/forums/topics/808012) by [SleepingD](/users/9612894)
```

Reply:
I think this will make it harder to read. Modes vary, so titles of the skin will be misaligned.

Reply:
true, but currently it's more difficult to tell at a glance which game mode(s) a skin supports (which may or may not be more important? not sure)

alternative solution: use some sort of "blank" or grayed-out mode icon to represent the ones that are missing, so they are all aligned still

Reply:
if you wan't to be able to identify the mode more easily then we shouldn't use lazer icons. They are quite ahrd to identify and I always mix up taiko and mania.

And like I said in a previous comment, I can't really change the order of things or amke a table out of everything, because the tesxt is automated by a programm a friend wrote